sbndqm/dqmAnalysis/CRT/SBND/BernCRTdqmSBND_module.cc

```diff
@@ -413,94 +487,197 @@ if(!fragmentHandle.isValid() || fragmentHandle->size() == 0)
+     * \param mode The MetricMode that the metric should operate in. Options are:
+     *    LastPoint: Every reporting_interval, the latest metric value is sent (For run/event numbers, etc)
+     *    Accumulate: Every reporting_interval, the sum of all metric values since the last report is sent (for counters)
+     *    Average: Every reporting_interval, the average of all metric values since the last report is sent (for rates)
 	
-    * We note that the level here is used as a threshold; we send any value of the metric above the level. For the CRT DQM, all levels = 0.
-    */
-
-    sbndaq::sendMetric("CRT_board", readout_number_str, "MaxADCValue", max, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "MaxADCChannel", maxindex + 32 * mac5, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "Flag", thisflag, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "baseline", baseline, 0, artdaq::MetricMode::Average);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "TS0", ts0, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "TS1", ts1, 0, artdaq::MetricMode::LastPoint);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "Deadtime", deadtime, 0, artdaq::MetricMode::Minimum);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "MissingT0", missingT0, 0, artdaq::MetricMode::Maximum);
-    sbndaq::sendMetric("CRT_board", readout_number_str, "MissingT1", missingT1, 0, artdaq::MetricMode::Maximum);
+     * We note that the level here is used as a threshold; we send any value of the metric above the level. For the CRT DQM, all levels = 0.
+     */
+
+    sbndaq::sendMetric("CRT_board", mac5_str, "Flag", thisflag, 0, artdaq::MetricMode::LastPoint);
+    sbndaq::sendMetric("CRT_board", mac5_str, "TS0", static_cast<int>(ts0), 0, artdaq::MetricMode::LastPoint);
+    sbndaq::sendMetric("CRT_board", mac5_str, "TS1", static_cast<int>(ts1), 0, artdaq::MetricMode::LastPoint);
+    sbndaq::sendMetric("CRT_board", mac5_str, "MissingT0", missingT0[mac5], 0, artdaq::MetricMode::Maximum);
+    sbndaq::sendMetric("CRT_board", mac5_str, "MissingT1", missingT1[mac5], 0, artdaq::MetricMode::Maximum);
  
     //only send clockdrift info when it makes sense to do so; that is, for T0 reset events.
-    if(isTS0 && isTS0good) {sbndaq::sendMetric("CRT_board", readout_number_str, "T0clockdrift", ts0 - 1e9, 0, artdaq::MetricMode::LastPoint);}
-    if(isTS1 && isTS1good) {sbndaq::sendMetric("CRT_board", readout_number_str, "T1clockdrift", ts1 - 1e9, 0, artdaq::MetricMode::LastPoint);}
+    if(isTS0 && isTS0good) {sbndaq::sendMetric("CRT_board", mac5_str, "T0clockdrift", static_cast<int>(ts0) - 1e9, 0, artdaq::MetricMode::LastPoint);}
+    if(isTS1 && isTS1good) {sbndaq::sendMetric("CRT_board", mac5_str, "T1clockdrift", ts1 - 1e9, 0, artdaq::MetricMode::LastPoint);}
```

Comment:
This is another metric that makes no sense and I deliberately removed.

Reply:
keeping the T1clockdrift metric as a comment for ease of (re)implementation